### ISSUE TITLE
GeoKrety: search trackables with secret trackingCode not working (any more?)

### ISSUE THREAD

**Issue by eddiemuc** [cgeo/cgeo#15143]
While refactoring trackable logging I played around with GeoKrety for the first time.

I noticed that searching a GeoKrety using its secret tracking code does not work in c:geo. Is this a bug or is this intended behaviour?

I found code in c:geo indicating that at some point in the past such a search would work. 
It seems c:geo tries to get the geocode for a trackable by calling the following URL: `https://api.geokrety.org/nr2id/{secretTrackingCode}`. However, this URL returns HTTP-404
(Reference to c:geo code: `GeokretyConnector.getGeocodeFromTrackingCode`)

**Comment by kumy** [comment]
:facepalm: rhaaaaaa!! sorry for that.
I've created https://github.com/geokrety/geokrety-website/issues/911 on GK side. (I will not have time to work on that until next week.)

**Comment by eddiemuc** [comment]
@kumy please don't maintain legacy endpoints just for the sake of c:geo. If there is another (preferred) way to search for geokrety by (secret) trackingCode we can use that as well. We just need to know how?

**Comment by kumy** [comment]
@eddiemuc I added the search by tracking code in the already used `export2` endpoint. It should land on staging in a few minutes and in the next few days on prod.

[documentation](https://new-theme.staging.geokrety.org/help/api#general_purpose_switches) has been updated:
- new parameter `tracking_code` has been added
- url were changed to the new ones
- parameters were changed to the new ones
- parameter `tz` has been documented
![image](https://github.com/cgeo/cgeo/assets/176794/b57890b0-ea07-4b95-bf7b-48bbb34024be)
![image](https://github.com/cgeo/cgeo/assets/176794/e359f60c-39e0-48d5-806a-55b292cfc547)

Let me know if you need something else :)

**Comment by bekuno** [comment]
Also we have then to remove the test:
~~~
java.lang.AssertionError: Failed to get response from trackable website geokrety.org proxy https://api.geokrety.org
at cgeo.watchdog.WatchdogTest.checkWebsite(WatchdogTest.java:105)
at cgeo.watchdog.WatchdogTest.testTrackableWebsites(WatchdogTest.java:115)
~~~

**Comment by eddiemuc** [comment]
Finally fixed with PR #15955

While fixing this I noticed that c:geo seems to use a totally outdated version of the geokrety-API. We need to update this someday...
